phpseclib/Net/SFTP.php

```diff
@@ -3086,7 +3086,7 @@ public function getSupportedVersions()
      * @return bool
      * @access private
      */
-    private function disconnect_helper($reason)
+    protected function disconnect_helper($reason)
```

Comment:
SFTP class extends SSH2 and overrides method disconnect_helper($reason), also it calls parent's method:

protected function disconnect_helper($reason)
    {
        $this->pwd = false;
        parent::disconnect_helper($reason);
    }

Parent's method was declared as private, so could not be called outside the class.
I changed it's visibility to protected. 
Also I have to change visibility of the successor's method.